packages/experimental/Badge/src/BadgeColorTokens.ts

```diff
@@ -224,13 +216,13 @@ export const defaultBadgeColorTokens: TokenSettings<BadgeTokens> = (t: Theme) =>
           {
             backgroundColor: globalTokens.color.grey[14],
             backgroundColorDark: globalTokens.color.white,
-            colorDark: globalTokens.color.grey[14],
+            colorDark: t.colors.neutralForeground1,
           },
           t,
         ),
       },
       outline: {
-        ...getOutlineColorProps({ color: globalTokens.color.grey[14], colorDark: globalTokens.color.white }, t),
+        ...getOutlineColorProps({ color: t.colors.neutralForeground1, colorDark: globalTokens.color.white }, t),
```

Comment:
Redlines say neutralForeground3 should be used.

Reply:
Also the border is specified is neutral stroke accessible

Reply:
Yeah, that's what I was talking about, it's easy to make a mistake when you need to check all the tables. Thanks for catching this